### PR TITLE
Region followers realization

### PR DESCRIPTION
**Pull request by Tali32**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/City/CreateCity/CreateCity.tsx [hunk 1]
```diff
@@ -45,15 +47,53 @@ import{
...
+
+  //const createNotification = async(userId : string, message : string) => {
+  //  await NotificationBoxApi.createNotifications(
+  //    [userId],
+  //    message + ": ",
+  //    NotificationBoxApi.NotificationTypes.UserNotifications,
+  //    `/cities/${id}`,
+  //    cityName
+  //    );
+  //}
```
Comment (christykobyletska) [inline src/pages/City/CreateCity/CreateCity.tsx:79]: Do you need these comments?

File: src/pages/Regions/Region.tsx [hunk 2]
```diff
@@ -202,13 +213,49 @@ const Region = () => {
...
+  *    cancelText: "Скасувати",
+  *    maskClosable: true,
+  *    onOk() {
+  *      {
+  *        //await handleSubmit(follower);
+  *      }
+  *    },
+  *  });
+  }*/
+
```
Comment (christykobyletska) [inline src/pages/Regions/Region.tsx:250]: and these comments?

File: src/pages/Regions/Region.tsx [hunk 3]
```diff
@@ -202,13 +213,49 @@ const Region = () => {
...
   const getRegion = async () => {
     setLoading(true);
 
     try {
       const response = await getRegionById(id);
       const response1 = await getRegionAdministration(id);
       const responce2 = await cityNameOfApprovedMember(userApi.getActiveUserId());
+      const response3 = await getRegionFollowers(id);
```
Comment (christykobyletska) [inline src/pages/Regions/Region.tsx:258]: Please, provide an appropriate name
This name doesn't meet with conventions

File: src/pages/Regions/RegionFollowers.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    const [regionName, setRegionName] = useState<string>("");
+    const [followers, setFollowers] = useState<RegionFollower[]>([]);
+    const [activeUserRoles, setActiveUserRoles] = useState<string[]>([]);
+    const [isActiveUserRegionAdmin, setIsActiveUserRegionAdmin] = useState<boolean>(false);
+
+    const getFollowers = async () => {
+      setLoading(true);
+      const response = await getRegionById(id)
+      const response1 = await getRegionFollowers(id);
+      const response2 = await getRegionAdministration(id);
```
Comment (christykobyletska) [inline src/pages/Regions/RegionFollowers.tsx:34]: Please, provide an appropriate name
This name doesn't meet with conventions

File: src/pages/Regions/RegionFollowers.tsx [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    
+    //const createNotification = async(userId : string, message : string) => {
+    //  await NotificationBoxApi.createNotifications(
+    //    [userId],
+    //    message + ": ",
+    //    NotificationBoxApi.NotificationTypes.UserNotifications,
+    //    `/regions/${id}`,
+    //    regionName
+    //    );
+    //}
```
Comment (christykobyletska) [inline src/pages/Regions/RegionFollowers.tsx:72]: Do you need these comments?

File: src/pages/Regions/RegionFollowers.tsx [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+    const deleteFollower = async (follower: RegionFollower) => {
+      await removeFollower(follower.id);
+      //await createNotification(follower.userId, `Нажаль, вашу заяву на створення станиці ${follower.cityName} відхилено. Для з'ясування причин відмови зверніться до голови округу`);
```
Comment (christykobyletska) [inline src/pages/Regions/RegionFollowers.tsx:76]: and these comment?

File: src/pages/Regions/RegionFollowers.tsx (hunk 5 above)
Comment (Tali32) [reply src/pages/Regions/RegionFollowers.tsx:72]: They will be realized in the next commit

File: src/pages/Regions/RegionFollowers.tsx (hunk 6 above)
Comment (Tali32) [reply src/pages/Regions/RegionFollowers.tsx:76]: They will be realized in the next commit

File: src/pages/Regions/RegionFollowers.tsx (hunk 5 above)
Comment (christykobyletska) [reply src/pages/Regions/RegionFollowers.tsx:72]: then ok

File: src/pages/Regions/RegionFollowers.tsx (hunk 4 above)
Comment (Tali32) [reply src/pages/Regions/RegionFollowers.tsx:34]: Will fix it with the next commit

File: src/pages/Regions/Region.tsx (hunk 3 above)
Comment (Tali32) [reply src/pages/Regions/Region.tsx:258]: Will fix it with the next commit

File: src/pages/City/CreateCity/CreateCity.tsx (hunk 1 above)
Comment (Tali32) [reply src/pages/City/CreateCity/CreateCity.tsx:79]: They will be realized in the next commit

File: src/pages/Regions/Region.tsx (hunk 2 above)
Comment (Tali32) [reply src/pages/Regions/Region.tsx:250]: They will be realized in the next commit

**Review by christykobyletska — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Review by sharix4 — APPROVED** [review]

**Pull request merged**